src/entry.js

```diff
@@ -1 +1,3 @@
-module.exports = require('./operations')
+import operations from './operations'
+
+module.exports = operations
```

Comment:
Sure, that makes sense as a separate refactor stage.

We won't have actual ES Module support until that `module.exports` has disappeared, though.